Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -79,7 +87,8 @@ final class CrudTests: MongoSwiftTestCase {
 private struct CrudTestFile: Decodable {
     let data: [Document]
     let testDocs: [Document]
-    var tests: [CrudTest] { return try! testDocs.map { try makeCrudTest($0) } }
+    var tests: [CrudTest] { return try!
+        testDocs.filter { ($0["operation"] as? Document)?["name"] as? String != "count" }.map { try makeCrudTest($0) } }
```

Comment:
this can probably be left as-is, and instead we just skip the test above in `doTests`. That way all the test skipping is centralized.

Reply:
doing that would require that we keep around the `CountTest` type as a placeholder to parse into. the files mix count and countDocuments so we can't skip a whole file, either.

Reply:
Oh, good point. I actually personally prefer the placeholder approach. Imo we should be able to deserialize all the tests, regardless of which ones we're going to run, and the testing logic should decide which ones get skipped. It feels a little weird here to be inspecting the documents directly right before we deserialize them. 

This isn't a huge deal of course, but the placeholder approach is the one I'm taking for the retryable reads tests, so I figured it was worth mentioning here.

Reply:
that's fine with me, switched to using a placeholder.